get_github_coderabbit_review/coderabbit_fetcher/async_processing/async_orchestrator.py

```
@@ -0,0 +1,447 @@
+                        commits = []
+
+            return {
+                'comments': comments or [],
+                'reviews': reviews or [],
+                'files': files or [],
+                'commits': commits or [],
+                'pr_info': pr_data['pr_info']
+            }
+
+        except asyncio.TimeoutError:
+            logger.error("Timeout while fetching GitHub data")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)